scripts/create-releasenotes.sh

```diff
@@ -4,5 +4,9 @@
 #
 # Usage: create-releasenotes.sh pr_num pr_url pr_title commit_id commit_url
 
-sed "s/PRNUM/$1/;s/PRURL/$2/;s/PRTITLE/$3/;s/COMMITID/$4/;s/COMMITURL/$5/" < gh_release_notes.in > gh_release_notes
+prurl=$(sed -e 's/[&\\/]/\\&/g; s/$/\\/' -e '$s/\\$//' <<< "$2")
+prtitle=$(sed -e 's/[&\\/]/\\&/g; s/$/\\/' -e '$s/\\$//' <<< "$3")
+commiturl=$(sed -e 's/[&\\/]/\\&/g; s/$/\\/' -e '$s/\\$//' <<< "$5")
```

Comment:
I've been doing some googling, and there are many other things that we'd have to quote. sed is a bit of a mess.
awk would be much easier... using `sub(pattern, input)` you only need to escape `\` and`&`...